README.md

```diff
@@ -548,16 +548,18 @@ ort:
 
 ### AWS S3 Storage
 
-An AWS S3 Bucket can be used to store scan results. You must provide a previously created Bucket, an optional AWS region
-(if not present it will use [default](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/region-selection.html))
-and credentials (if not provided, system [default](https://docs.aws.amazon.com/sdk-for-kotlin/latest/developer-guide/credential-providers.html)
-values will be used). Optionally, you can set the data to be stored compressed.
+An [AWS S3 Bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html) can be used to store scan
```

Comment:
Again, move the hunks in this commit to the commits that introduced the original respective code.

Reply:
Addressed by squashing